pkgs/development/libraries/libyuv/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchgit
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  name = "libyuv";
+
+  version = "1787";
+
+  src = fetchgit {
+    url = "https://chromium.googlesource.com/libyuv/libyuv.git";
+    rev = "refs/heads/stable";
+    sha256 = "sha256-DtRYoaAXb9ZD2OLiKbzKzH5vzuu+Lzu4eHaDgPB9hjU=";
+  };
+
+  patches = [ ./pkg-config.patch ];
```

Comment:
Have you attempted to upstream this patch? If not, i think we should. Also, a comment explaining why we need it might be nice.